src/swapnali/StringOper7.java

```diff
@@ -0,0 +1,35 @@
+package swapnali;
+
+class StringOper7{
+		
+	void printeven(String str){
+			for(int index=0;index<str.length();index++)
+				if(index%2==0)
+				{
+					System.out.println(str.charAt(index));
+				}
+		}
+		
+		void printvowels(String str){
+			
+			for(int index=0;index<str.length();index++) {
+				
+				char vowels[]={'a','e','i','o','u'};
+				for(int count=0;count<vowels.length;count++)
```

Comment:
instead of two for loop, you can handle this in a single loop

Take a look of below code & compare
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/81/files